### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by glatard**

### CODE REVIEW

**Comment by ttaa9** [comment]
Seems good. I assume you made sure the tool still works after the indentation changes.

File: tools/localExec.py [hunk 1]
```diff
@@ -487,7 +486,7 @@ def errExit(msg, print_usage = True):
     if print_usage: parser.print_usage()
     sys.stderr.write('Error: ' + msg + '\n')
     sys.exit(1)
-  given = lambda x: not (x is None)
+    given = lambda x: not (x is None)
```
Comment (ttaa9) [inline tools/localExec.py]: I'm not sure if this should be indented (?)

File: tools/README.md [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+    * `bundle install`
+  * Usage:
+    * `validator.rb ../schema/descriptor.schema.json ./tool.json`
+  * Usage example with Docker image:
+    * `docker run --rm -v $PWD:/work -w /work boutiques/boutiques validator.rb /usr/local/boutiques/schema/descriptor.schema.json ./tool.json`
+
+2. `localExec.py`: a Python script to test inputs on Boutiques JSON descriptors.
+  * Usage example:
+    * Generates 4 random command lines: `docker run --rm -v $PWD:/work -w /work boutiques/boutiques localExec.py -n 4 -r ./tool.json`
+  * Check the help page (`-h` option) for more options and documentation.
```
Comment (ttaa9) [inline tools/README.md]: It is not necessary but if you wanted maybe we could add something like:
* Execute with an input file of parameters: 
`docker run --rm -v $PWD:/work -w /work boutiques/boutiques localExec.py -i input_param_file.json -e ./tool.json`

or

* Print a command line based on parameters from an input file: 
`docker run --rm -v $PWD:/work -w /work boutiques/boutiques localExec.py -i input_param_file.csv ./tool.json`

**Comment by glatard** [comment]
@ttaa9 thanks for the useful comments!

**Pull request merged**